lhapdf_pdfsetsindex.file

```diff
@@ -93,7 +93,7 @@
 13070 CT14nnlo_as_0121 1
 13071 CT14nnlo_as_0122 1
 13072 CT14nnlo_as_0123 1
-13081 CT14nnloIC 1
+13081 CT14nnloIC 2
```

Comment:
@mkirsano could you please clarify the meaning of this (and following) changes? What does this last digit imply?

Reply:
The last digit is the "data version" https://lhapdf.hepforge.org/pdfsets.html

Reply:
@mkirsano thank you, sorry to be pedantic, but was it verified that the new versions are identical to the old one, for PDFs that were already present?

Reply:
For this particular change one family was added to the set CT14nnloIC, now there are 6 families instead of 5. A hint that the information is only added. It was not verified one number by one.